.github/workflows/federated-integ-test.yml

```diff
@@ -0,0 +1,201 @@
+---
+
+name: Federated integ test
+
+on:  # yamllint disable-line rule:truthy
+  workflow_dispatch:
+  workflow_run:
+    workflows:
+      - build-all
+    branches:
+      - master
+      - 'v1.*'
+    types:
+      - completed
+  push:
+    branches:
+      - master
+      - 'v1.*'
+  pull_request:
```

Comment:
I guess this has to go before we merge this

Reply:
yep, this si gone now